checkpoint_windows.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"github.com/codegangsta/cli"
+)
+
+var checkpointCommand = cli.Command{}
+
+//func getCheckpointImagePath(context *cli.Context) string {
+//return ""
+//}
```

Comment:
Yeah, was a hang-over from when I was trying to get this to compile (Windows won't support checkpoints). Removed.